README.md

```diff
@@ -25,10 +25,14 @@ function handleTransition() {
   document.documentTransition.prepare({
     rootTransition: "reveal-left",
   }).then(() => {
-    changeBodyBackground();
+    await changeBodyBackground();
     document.documentTransition.start().then(() => console.log("transition finished"));
   });
 }
+
+async function changeBodyBackground() {
```

Comment:
This redefines a function defined above. I'm also not clear what the effective change is here?

await changeBodyBackground(); with async changeBodyBackground
vs
changeBodyBackground() with non-async changeBodyBackground.

Is there a difference in this context?

Reply:
I added another section to explain the async stuff. Better than complicating the base case.